pheno005/dataset_description.json

```diff
@@ -0,0 +1,12 @@
+{
+	"Name": "BEP036 example dataset 05: Two multi-run sessions with one imaging-only session.",
+	"BIDSVersion": "1.9.0",
```

Comment:
Should this be the latest BIDSVersion, 1.10.0?

Reply:
changed!